kerchunk/tests/test_hdf.py

```diff
@@ -1,31 +1,50 @@
+import asyncio
 import fsspec
 import os.path as osp
 
+import fsspec.implementations
+import fsspec.implementations.reference
+
 import kerchunk.hdf
 import numpy as np
 import pytest
 import xarray as xr
 import zarr
-import h5py
+
+from packaging.version import Version
```

Comment:
We could have a global `is_zarr3` somewhere to avoid having to make the comparison in multiple places.